sdk/monitor/azure-opentelemetry-exporter-azuremonitor/samples/metrics/observer.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import psutil
```

Comment:
All the samples need docstrings - please create an issue to make sure we have it tracked :)

Reply:
Will create an issue.

Reply:
https://github.com/Azure/azure-sdk-for-python/issues/16382